client/lib/i18n-utils/guess-timezone.ts

```diff
@@ -0,0 +1,91 @@
+// This list comes from https://github.com/eggert/tz/blob/master/backward
+const linkedTimezones = {
```

Comment:
Would it make sense to export this data from [this file](https://github.com/Automattic/wp-calypso/blob/master/client/lib/i18n-utils/guess-timezone.js#L17) and import it here instead? This add a redundant ~1KB to the uncompressed bundle.